Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -2142,6 +2245,55 @@ var stagger = function(
   }));
 };
 
+type LoopAnimationConfig = AnimationConfig & { iterations: number };
+
+var loop = function(
+  animation: CompositeAnimation,
+  config: LoopAnimationConfig,
+): CompositeAnimation {
+  if (!config) config = {}; // Use empty config to avoid checking if (config && ...) every time
```

Comment:
You could use default params + descructuring to make this nicer

`{iterations = -1}: LoopAnimationConfig = {}`